src/components/CustomFeesEIP1559.vue

```diff
@@ -339,29 +339,29 @@ export default {
     },
     minerTipFiat() {
       const fiat = prettyFiatBalance(
-        getSendFee(this.nativeAsset, this.minerTip),
+        getSendFee(this.asset, this.minerTip),
         this.fiatRates[this.nativeAsset]
       )
       return isNaN(fiat) ? 0 : fiat
     },
     maxFiat() {
       const fiat = prettyFiatBalance(
-        getSendFee(this.nativeAsset, this.maximumFee),
+        getSendFee(this.asset, this.maximumFee),
         this.fiatRates[this.nativeAsset]
       )
       return isNaN(fiat) ? 0 : fiat
     },
     minimum() {
       const minimumFee = this.minerTip + this.suggestedBaseFeePerGas
-      const totalMinFee = getSendFee(this.nativeAsset, minimumFee).plus(this.totalFees.slow)
+      const totalMinFee = getSendFee(this.asset, minimumFee).plus(this.totalFees.slow)
       return {
         amount: BN(totalMinFee).dp(6),
         fiat: prettyFiatBalance(this.totalFees.slow, this.fiatRates[this.nativeAsset])
       }
     },
     maximum() {
       const maximumFee = this.maximumFee
-      const totalMaxFee = getSendFee(this.nativeAsset, maximumFee).plus(this.totalFees.fast)
+      const totalMaxFee = getSendFee(this.asset, maximumFee).plus(this.totalFees.fast)
```

Comment:
I guess this is done like this, because if you send/swap ERC20 maximum is the maximum balance of your erc20 + you need nativeAsset to pay the fee. With this change is it taking in mind this logic and behave the same way for erc20 and native assets as before